jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -77,6 +80,27 @@ def test_nirspec_parser(mosviz_helper, tmp_path, instrument_arg):
                                              data_label=data_label)
 
 
+@pytest.mark.filterwarnings('ignore')
+@pytest.mark.remote_data
+def test_nirspec_level2_parser(mosviz_helper, tmp_path):
+    '''
+    Tests loading our default MosvizExample notebook data
+    Also tests no instrument keyword fallback, and IntraRow linking
+    '''
+
+    test_data = 'https://stsci.box.com/shared/static/mytqf082lpbfia7wlwjq6p1h5cggd9h6.zip'
+    fn = download_file(test_data, cache=True, timeout=30)
```

Comment:
Why not 100 like the other one?
```suggestion
    fn = download_file(test_data, cache=True, timeout=100)
```

Reply:
Not sure why I made it different, honestly.